app.config.js

```diff
@@ -13,16 +13,10 @@ configurationFiles.push(
 )
 
 const isDrive = process.env.COZY_APP_SLUG === 'drive'
-const isPhotosServices = process.env.COZY_APP_SLUG === 'photos:services'
+const isPhotos = process.env.COZY_APP_SLUG === 'photos'
```

Comment:
```suggestion
const isPhotos = process.env.COZY_APP_SLUG === 'photos' || process.env.COZY_APP_SLUG === 'photos:services'
```

Reply:
I think we don't need the script `photos:services` anymore since we build services with the photos browser application

Reply:
Ok, fair enough